### PR TITLE
Esc 252 email retrieve

### PR DESCRIPTION
**Pull request by priyankadhiman3006**
ticket details -> https://jira.tools.tax.service.gov.uk/browse/ESC-252

### CODE REVIEW

File: app/uk/gov/hmrc/eusubsidycompliancefrontend/connectors/RetrieveEmailConnector.scala [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+}
+
+@Singleton
+class RetrieveEmailConnectorImpl @Inject()(http: HttpClient,
+                                           servicesConfig: ServicesConfig
+                                          )(implicit ec: ExecutionContext
+                                          ) extends RetrieveEmailConnector {
+
+  val logger: Logger = Logger(this.getClass)
```
Comment (sam-miele) [inline app/uk/gov/hmrc/eusubsidycompliancefrontend/connectors/RetrieveEmailConnector.scala:42]: Doesn't look like this is being used?

Comment (priyankadhiman3006) [reply app/uk/gov/hmrc/eusubsidycompliancefrontend/connectors/RetrieveEmailConnector.scala:42]: yup, will remove 👍

**Review by sam-miele — APPROVED** [review]

**Comment by sam-miele** [comment]
One NABD - not a blocker. But good to go otherwise.

File: app/uk/gov/hmrc/eusubsidycompliancefrontend/connectors/RetrieveEmailConnector.scala (hunk 1 above)
Comment (priyankadhiman3006) [reply app/uk/gov/hmrc/eusubsidycompliancefrontend/connectors/RetrieveEmailConnector.scala]: [updated](https://github.com/hmrc/eu-subsidy-compliance-frontend/pull/27/commits/1de6e48b8cfe03078a65f8f945dca870b1dc7f91)

**Pull request merged**